x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -488,6 +491,7 @@ public class Constants {
         "indices:admin/block/add[s]",
         "indices:admin/cache/clear",
         "indices:admin/data_stream/lazy_rollover",
+        "indices:admin/data_stream/reindex",
```

Comment:
Also, this one seems to be only added in 9.0 with a backport pending to 8.18+ (https://github.com/elastic/elasticsearch/pull/116780).

Reply:
Yeah just added yesterday. The backport to 8.18 is coming soon (https://github.com/elastic/elasticsearch/pull/117251).

Reply:
And it will not be backported to 8.17 or earlier.